spec/services/conversation_service/creator_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Services::ConversationService::Creator', type: :service do
+  subject { ConversationService::Creator.new(sender: sender, receivers: receivers, topic: topic) }
+
+  let(:topic) { create(:topic) }
+  let!(:sender) { create(:user) }
+  let!(:receivers) { create_list(:user, 3) }
+
+  it 'creates conversations between sender and receivers' do
+    expect { subject.call }.to change(Conversation, :count).by(3)
+  end
```

Comment:
There should be tests for the case when there already exists a conversation between two users.

Reply:
@sebastiancaraballo This service is only in charge of creating the conversations.
The service that handles this scenario is compatible_target_finder service. 
There is a test that validates this case in spec/services/target_service/compatible_target_finder_spec.rb line 28.